.gitignore

```diff
@@ -10,6 +10,7 @@ build_android/.*
 build_android/*.apk
 build_android/Play.iml
 build_android/projectFilesBackup/
+build_ios/build
```

Comment:
Not sure why directory deletions never seem to sync. I have the new folders, but also the old ones.